pkg/client/options.go

```diff
@@ -154,6 +154,23 @@ func (f FieldOwner) ApplyToUpdate(opts *UpdateOptions) {
 	opts.FieldManager = string(f)
 }
 
+// ApplyToSubResourcePatch applies this configuration to the given patch options.
+func (f FieldOwner) ApplyToSubResourcePatch(opts *SubResourcePatchOptions) {
```

Comment:
Please add tests

Reply:
done.

Since I couldn't find any tests for `FieldOwner`, I wrote some from scratch. I hope they're to your taste.